genai/generativelanguagepb_veneer.gen.go

```diff
@@ -835,3 +1004,80 @@ func (v TaskType) String() string {
 	}
 	return fmt.Sprintf("TaskType(%d)", v)
 }
+
+// Tool details that the model may use to generate response.
+//
+// A `Tool` is a piece of code that enables the system to interact with
+// external systems to perform an action, or set of actions, outside of
+// knowledge and scope of the model.
+type Tool struct {
+	// Optional. A list of `FunctionDeclarations` available to the model that can
+	// be used for function calling.
+	//
+	// The model or system does not execute the function. Instead the defined
+	// function may be returned as a [FunctionCall][content.part.function_call]
+	// with arguments to the client side for execution. The model may decide to
+	// call a subset of these functions by populating
+	// [FunctionCall][content.part.function_call] in the response. The next
```

Comment:
Nit: these links in comments don't render properly (see https://pkg.go.dev/cloud.google.com/go/vertexai@v0.7.1/genai#Tool for the vertex equivalent)

They seem to come from the protos -- I wonder if it's possible to fix them up? (in protoveneer, to affect both SDKs)

Reply:
As you say, this is a protoveneer change, so it will be a separate PR. I'll put it on my TODO list.